packages/stream_chat_flutter/lib/src/message_list_view/message_list_view.dart

```diff
@@ -1309,62 +1148,6 @@ class _StreamMessageListViewState extends State<StreamMessageListView> {
     return child;
   }
 
-  StreamSubscription? _messageNewListener;
-
-  @override
-  void initState() {
-    _scrollController = widget.scrollController ?? ItemScrollController();
-    _itemPositionListener =
-        widget.itemPositionListener ?? ItemPositionsListener.create();
-
-    _getOnThreadTap();
-    super.initState();
-  }
-
-  @override
-  void didChangeDependencies() {
-    final newStreamChannel = StreamChannel.of(context);
-    _streamTheme = StreamChatTheme.of(context);
-    _userPermissions = newStreamChannel.channel.ownCapabilities;
```

Comment:
@GroovinChip

Reply:
What specifically is missing? The `_userPermissions = newStreamChannel.channel.ownCapabilities;` line?

Reply:
Fixed in 4247dde